lib/common/network/fake_api_manager.dart

```diff
@@ -14,8 +13,7 @@ enum FakeApi {
 
 class FakeApiManager {
   static final _enabledFakeApi = <FakeApi>{
-    //Place here fake api that should be enabled by default
-    // if (kDebugMode) FakeApi.characters,
+    if (kDebugMode) FakeApi.characters,
```

Comment:
let's revert this changes, app in develop and master branch should work with real api (but add character feature won't work due to api luck)

Reply:
done)